installer.tpl.sh

```diff
@@ -20,7 +20,7 @@ unsupported_os ()
 detect_os ()
 {
   if [[ ( -z "${os}" ) && ( -z "${dist}" ) ]]; then
-    if [ -e /etc/centos-release ]; then
+    if [ -e /etc/centos-release ] && [ -z "$(grep "RED OS" /etc/centos-release)" ]; then
```

Comment:
I think we should detect `os`/`dist` with a similar way for CentOS:

```bash
if [ -e /etc/centos-release ]; then
  if [ -n "$(grep "RED OS" /etc/centos-release)" ]; then
    os="redos"
    dist=$(grep -Po "[6-9].[0-9]" /etc/centos-release)
  else
	os="centos"
    dist=$(grep -Po "[6-9]" /etc/centos-release | head -1)
  fi
...
```

Reply:
In You way variable set to 7 on 7.3.1, but in repo we use 7.3 as dist version.

Reply:
I decide using /etc/os-release file because we can get version in one step without preparation. And I separate RedOS from CentOS because It based on CentOS but Isn't same.

Reply:
fixed